code/modules/projectiles/guncrafting/attachments/bullets.dm

```diff
@@ -0,0 +1,46 @@
+/obj/item/weapon/gun_attachment/bullet
+	gun_type = CUSTOMIZABLE_PROJECTILE
+	uses_overlay = FALSE
+	icon_state = "bullet"
+	not_okay = /obj/item/weapon/gun_attachment/bullet
+	no_revolver = 0
+
+/obj/item/weapon/gun_attachment/bullet/polonium
+	name = "Polonium Rounds"
+
+/obj/item/weapon/gun_attachment/bullet/polonium/on_fire(var/obj/item/weapon/gun/owner, var/obj/item/projectile/bullet)
+	..()
+	bullet.irradiate += 20
```

Comment:
While the damage is already halved, irradiating a target is actually pretty severe. So I recommend taking it down to:

bullet.irradiate += 10

You will afterall shoot a guy several times, but 20 is just a tad bit to much.